Cabal/doc/developing-packages.rst

```diff
@@ -1336,6 +1336,10 @@ The following flags are supported by the ``outdated`` command:
 ``--new-freeze-file``
     Read dependency version bounds from the new-style freeze file
     (``cabal.project.freeze``) instead of the package description file.
+``--project-file`` *PROJECTFILE*
```

Comment:
How does this interact w/ the `--new-freeze-file` flag?

Reply:
Last one wins (or it should be). I'll add a test and make sure it's explicitly mentioned in the docs.